wgpu-hal/src/gles/device.rs

```diff
@@ -110,22 +110,22 @@ impl super::Device {
     ///
     /// - `name` must be created respecting `desc`
     /// - `name` must be a texture
-    /// - If `drop_guard` is [`None`], wgpu-hal will take ownership of the texture. If `drop_guard` is
-    ///   [`Some`], the texture must be valid until the drop implementation
+    /// - If `drop_callback` is [`None`], wgpu-hal will take ownership of the texture. If
+    ///   `drop_callback` is [`Some`], the texture must be valid until the drop implementation
```

Comment:
This comment is out of date

Reply:
I don't really understand why or what to change it to ;(

Reply:
can you elaborate?

Reply:
It refers to the drop implementation of the drop guard, as opposed to the new callback api

Reply:
Oh, sure, I missed that